variables.tf

```diff
@@ -294,3 +294,9 @@ variable "kms_key_id" {
   default     = null
   description = "KMS key ID used to encrypt EBS volume. When specifying kms_key_id, ebs_volume_encrypted needs to be set to true"
 }
+
+variable "volume_tags_enabled" {
+  type        = bool
+  default     = false # To preserve existing behaviour
```

Comment:
```suggestion
  default     = true
```

Reply:
@gregnphe Adding tags to volumes by default seems like good behavior for the default. Let's move folks forward without having them to opt in since it shouldn't hurt them. Make this change and we'll get this merged 👍